README.md

```diff
@@ -1,140 +1,32 @@
-  - [Version policy](#version-policy)
-  - [Browser support](#browser-support)
-  - [Right-to-left support](#right-to-left-support)
-  - [Server-side rendering](#server-side-rendering)
-  - [Advanced usage](#advanced-usage)
-- [Contribute to Fluent UI React](#contribute-to-fluent-ui-react)
-- [Building the repo](#building-the-repo)
-  - [Testing](#testing)
-  - [Advanced building tips](#advanced-building-tips)
-- [Licenses](#licenses)
-- [Changelog](#changelog)
-
-## Using Fluent UI React
-
-### Creating a new app
-
-To create a simple React app using [Create React App](https://create-react-app.dev), [install Node.js](https://nodejs.org), then run:
-
-```sh
-npx create-react-app my-app
-cd my-app
-npm install @fluentui/react
-npm start
-```
-
-See the next section for some starter code using Fluent UI React controls.
-
-### Integrating in your project
-
-```sh
-npm i @fluentui/react
-
-# Or, use yarn
-yarn add @fluentui/react
-```
-
-This will add the package as a dependency in your `package.json` file and download it under `node_modules/@fluentui/react`.
-
-The library includes ES2015 module entry points under the `lib` folder (use `lib-amd` if you need AMD, or `lib-commonjs` if you need commonjs). To use a control, import it and then use it in your render method:
-
-```jsx
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { PrimaryButton } from '@fluentui/react/lib/Button';
+Fluent UI web represents a collection of utilities, React components, and web components for building web applications.
 
-ReactDOM.render(<PrimaryButton>I am a button.</PrimaryButton>, document.getElementById('root'));
-```
+The Fluent UI web repository is home to 3 main projects today, the following table will help navigate them and understand their differences.
 
-### Version policy
+<!-- prettier-ignore-start -->
+|   | React | React Northstar | Web Components |
+|---| ----- | --------------- | -------------- |
+| **Overview**    | Mature, refreshing with new concepts from react-northstar. | Newer, has concepts we're iterating on. | Web Component implementation of Fluent UI. |
+| **Used By**     | Office| Teams | TODO |
+| **Read Me**     | [README.md](https://github.com/microsoft/fluentui/tree/master/packages/office-ui-fabric-react/README.md) | [README.md](https://github.com/microsoft/fluentui/tree/master/packages/fluentui/README.md) | [README.md](https://github.com/microsoft/fluentui/tree/master/packages/web-components/README.md) |
+| **Repo**        | [./packages/office-ui-fabric-react](https://github.com/microsoft/fluentui/tree/master/packages/office-ui-fabric-react) | [./packages/fluentui/react-northstar](https://github.com/microsoft/fluentui/tree/master/packages/fluentui/react-northstar) | [./packages/web-components](https://github.com/microsoft/fluentui/tree/master/packages/web-components) |
+| **Quick Start** | Quick Start | [Quick Start](https://fluentsite.z22.web.core.windows.net/quick-start) | Quick Start |
```

Comment:
@chrisdholt Also, do you all have a good quick start link for @fluentui/web-components?
I found this: https://www.fast.design/docs/components/getting-started/ but nothing for @fluentui/web-components specifically.

Reply:
That's part of us trying to onboard with the documentation as it is right now - I don't know that us hosting Fluent specific documentation for the controls (even though the API's are extremely similar if not the same) makes sense on the FAST site. For the quick start though, I do think it makes sense to point to the FAST documentations for *using FAST* in and of itself. For the quick start, what are we looking for there? I can point to the best current place and we can perhaps massage the messaging a bit if need be.

Reply:
It's just the basic getting starting info. 

Examples: 
- https://fluentsite.z22.web.core.windows.net/quick-start
- https://developer.microsoft.com/en-us/fluentui#/get-started/web

For web-components for now we could just say See README.md and link to your readme, or if you have another link that's fine.

Reply:
Something like this: 
`[See README.md](https://github.com/microsoft/fluentui/tree/master/packages/web-components/README.md)`